apps/server/src/server.module.ts

```diff
@@ -109,3 +116,21 @@ export class ServerTestModule {
 		};
 	}
 }
+
+@Module({
+	imports: [
+		MikroOrmModule.forRoot({
+			...defaultMikroOrmOptions,
+			type: 'mongo',
+			// TODO add mongoose options as mongo options (see database.js)
+			clientUrl: DB_URL,
+			password: DB_PASSWORD,
+			user: DB_USERNAME,
+			entities: ALL_ENTITIES,
+			allowGlobalContext: true,
+			// debug: true, // use it for locally debugging of querys
+		}),
+	],
+	controllers: [ServerController],
+})
+export class ServerFeathersTestModule {}
```

Comment:
This test module is not necessary anymore as discussed earlier.
This remark is just a reminder to apply the patch.